WTWSMS/common/religions/00_religions.txt

```diff
@@ -4378,14 +4439,14 @@ pagan_group = {
 		heresy_icon = 85
 		
 		color = { 0.5 0.05 0.0 }
-		#parent = hellenic_pagan
+		parent = hellenic_pagan

```

Comment:
They ARE a Hellenic heresy, and were for most of this Mod's existence. They worship the SATURN, the GREECO-ROMAN god, and observe much of the Hellenic pantheon.